### PR TITLE
add more to paired t test & editorial changes

### PR DESCRIPTION
**Pull request by K3ra-y**
Added more details to paired sample t-test (assumptions to not violate)
Changed the wording on hypothesis
Deleted repetitive parts 'questions to consider' which are actually made into the main survey questions
Minor editorial changes

### CODE REVIEW

File: Milestone1.md [hunk 1]
```diff
@@ -32,26 +32,26 @@
...
- Additionally, we would like to know if the video about recycling *changes* their opinion on the importance of sustainability.
+ Additionally, we would like to know if the video about recycling *changes* their opinion on the importance of sustainability and if so which predictors plays role in this change.
 
  ### Plan for Analyzing the Survey Results
 
- **Null hypothesis**: A person’s frequency of recycling does not influence her/his opinion and attitudes towards the importance of sustainability.
+ **Null hypothesis**: A persons opinion and attitudes towards the importance of sustainability is not affected by frequency of recycling, age, and whether they grow up in environmentally conscious family or not.
 
- **Alternative hypothesis**: A person’s frequency of recycling influences her/his opinion and attitudes towards the importance of sustainability.
+ **Alternative hypothesis**: A persons opinion and attitudes towards the importance of sustainability is affected by frequency of recycling, age or whether they grow up in environmentally conscious family or not.
```
Comment (mchiriboga) [inline Milestone1.md]: I think that we still need the 's here (_person's opinion_ instead of _persons opinion_)

File: Milestone1.md [hunk 2]
```diff
@@ -32,26 +32,26 @@
...
 
  ### Other question we aim to answer
 
- Additionally, we would like to know if the video about recycling *changes* their opinion on the importance of sustainability.
+ Additionally, we would like to know if the video about recycling *changes* their opinion on the importance of sustainability and if so which predictors plays role in this change.
 
  ### Plan for Analyzing the Survey Results
 
- **Null hypothesis**: A person’s frequency of recycling does not influence her/his opinion and attitudes towards the importance of sustainability.
+ **Null hypothesis**: A persons opinion and attitudes towards the importance of sustainability is not affected by frequency of recycling, age, and whether they grow up in environmentally conscious family or not.
```
Comment (mchiriboga) [inline Milestone1.md]: I think that we still need the 's here (_person's opinion_ instead of _persons opinion_)

**Review by mchiriboga — COMMENTED** [review]
Looks good, @K3ra-y!
Just 2 minor comments

**Comment by mchiriboga** [comment]
Thanks for updating, @K3ra-y! :)
I'll merge your PR

File: Milestone1.md (hunk 2 above)
Comment (K3ra-y) [reply Milestone1.md]: Thank you for pointing it out. I fixed the typo.

File: Milestone1.md (hunk 1 above)
Comment (K3ra-y) [reply Milestone1.md]: thank you for pointing out. I fixed the typo.

**Pull request merged**